src/index.js

```diff
@@ -57,6 +67,7 @@ io.on('connection', (socket) => {
 
     socket.on('getLocation', (location, callback) => {
         const user = getUser(socket.id)
+        getRooms()
```

Comment:
do you really need that here?

Reply:
Fixed.
(probably was testing something at some point)